file_open_menu

will-moore opened the pull request:
From comment on testing sheet
https://docs.google.com/spreadsheets/d/1qdOAv0wUgAVHEtlVroOwkeNUoNwst-APSPFJk6-3Sz8/edit#gid=0
"* File menu. When clicking on an item, a dialog pops up. To be consistent with Save a Copy...
All entries should have ... e.g. Open ..."

This PR adds the 3 dots to the commands which are incomplete (need other options chosen in a dialog). 
These are Open... and Import from JSON...

Also, improved layout of the drop-down menu options that have keyboard shortcuts, aligning the shortcut keys to the right of each menu.

To test:
 - Check the File and Edit menus looks OK on Mac
 - Check that on Windows the shortcut keys show ```Ctrl+N``` instead of apple key.

![screen shot 2018-04-23 at 12 35 05](https://user-images.githubusercontent.com/900055/39124288-d2b6bfb6-46f2-11e8-9476-ecc0786127aa.png)

pwalczysko commented:
Bug: On Mac Firefox, I have the shortcuts to the right of each line, but unfortunately, on a new line.
![screen shot 2018-04-25 at 13 51 40](https://user-images.githubusercontent.com/2478303/39246938-eae11110-488f-11e8-93e2-dba34e230682.png)

pwalczysko commented:
RFE: Unify capitalization of ``Save A Copy`` -> in line with ``Export as json``, where the ``as`` is lower case ``a``, the ``A`` in ``Save A Copy`` should be lower case too.

pwalczysko commented:
RFE: @will-moore mentioned that the capitalization of the word ``json`` is different from the common usage -> rectify this please.

mtbc commented:
The Mac detection seems rather broken: it's even showing the weird Mac four-loop symbol on Linux, both Firefox and Chrome. Maybe outwith this PR though.

mtbc commented:
Yes, Ctrl+ would be good for Linux: maybe detect Mac to show its weird thing and Ctrl otherwise.

mtbc commented:
From Linux got Ctrl+ now, great! :+1:

pwalczysko commented:
Works fine on Win7 IE11, FF and Chrome.
Works fine on mac FF, Chrome and Safari. 

Ready to merge FMPOV

will-moore commented:
Save doesn't open a dialog (unless previously unsaved) so doesn't need ```...```.
Added dots to Export as JSON... and fixed json -> JSON in dialog title and text.

jburel commented:
Looks good
Save is tricky since it will pop a dialog for unsaved figure.
Let's keep it as ``Save``.
Merging
